LICENSE

```diff
@@ -11,7 +11,7 @@ modification, are permitted provided that the following conditions are met:
   this list of conditions and the following disclaimer in the documentation
   and/or other materials provided with the distribution.
 
-* Neither the name of integrations-core nor the names of its
+* Neither the name of apigentools nor the names of its
```

Comment:
You can also set `the copyright holder`

Reply:
Not sure I understand, what should I change?

Reply:
`apigentools ` => `the copyright holder`